internal/controller/imagerepository_controller.go

```diff
@@ -290,7 +290,8 @@ func (r *ImageRepositoryReconciler) reconcile(ctx context.Context, sp *patch.Ser
 	// Scan the repository if it's scan time. No scan is a no-op reconciliation.
 	// The next scan time is not reset in case of no-op reconciliation.
 	if ok {
-		reconcile.ProgressiveStatus(false, obj, meta.ProgressingReason, "scanning: %s", reasonMsg)
+		drift := reasonMsg == scanReasonEmptyDatabase
```

Comment:
Add a comment here please explaining that this sets the Ready condition to Unknown when the db is empty